knx.py

```diff
@@ -14,6 +14,8 @@
 KNXWRITE = 0x80
 KNXREAD = 0x00
 
+DT_BIT = '1.*'
```

Comment:
Are these well-known KNX definitions or is this something arbitrary?o
Maybe it would make sense to define an [enum](https://docs.python.org/3/library/enum.html?highlight=enum#module-enum) since this is something that's exposed to users and that would clarify the allowed values a bit better.